manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -551,6 +551,96 @@ The expected values are:
             - type: clear
               field: associations
 
+Calculate
+_________
+
+This action allows to calculate numeric attribute values, with simple mathematical operations (addition, substraction, multiplication and division)
+
+This action only accepts number, measurement or price collection attributes for both source and destination.
+
+The action is split in 3 distinct parts:
+
+**destination**: the value you want to update. It can be composed of :
+
+- field: the attribute code (required)
+- locale: locale code if the attribute is localizable (optional)
+- scope: channel code if the attribute is scopable (optional)
+- currency: currency code if the attribute is a price collection (optional)
+- unit: unit code if the attribute is a measurement (optional, the default measurement unit of the attribute will be used if not set)
+
+**source**: the first operand of the operation. It can be composed of either:
+
+- value: a constant value (number, for instance 3.14)
+
+or:
+
+- field: attribute code of the source value (required)
+- locale: locale code if the attribute is localizable (optional)
+- scope: channel code if the attribute is scopable (optional)
+- currency: currency code if the attribute is a price collection (optional)
+
+**operation_list**: the list of operations to execute (obviously, at least one operation is required)
+
+It is exactly the same format as the ``source`` property, plus one required field:
+
+- operator: can be either ``add``, ``substract``, ``multiply`` or ``divide`` (required)
+
+    .. warning::
+
+       The operations are applied in the order they are provided, regardless of any mathematical priority.
+       For instance, 5 - 3 + 2 x 5 will result in ((5 - 3) + 2) x 5) = 20
+
+If a product value required in an operation is empty, or if a division by zero occurs, the product won't be updated.
+
+.. tip::
+
+    For instance, in order to calculate the volume of a cone, given a radius and a height, you can use the following action:
```

Comment:
You should write the mathematical action here, like volumne = PI * radius * radius * height / 3